src/counter_analysis_toolkit/dcache.c

```diff
@@ -11,12 +11,14 @@ static void print_core_affinities(FILE *ofp);
 
 extern char* eventname;
 
-int min_size, max_size, is_core = 0;
+long min_size, max_size;
```

Comment:
Maybe better use `int64_t` or `long long`. On some system `long` is same size as `int`

Reply:
Thank you for pointing this out. I will update it to 'long long.'

Reply:
This has been updated per your suggestion.